modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -437,6 +441,21 @@ private void ApplyNecessaryChangesToSolution()
             }
         }
 
+        private void UpdateLastValidBuildDateTime()
+        {
+            var dllName = $"{GodotSharpDirs.ProjectAssemblyName}.dll";
+            var path = GodotSharpDirs.ProjectBaseOutputPath.PathJoin("Debug").PathJoin(dllName);
```

Comment:
Since this is a normal path, we should probably use BCL methods to join them.

```suggestion
            string dllName = $"{GodotSharpDirs.ProjectAssemblyName}.dll";
            string path = Path.Combine(GodotSharpDirs.ProjectBaseOutputPath, "Debug", dllName);
```